src/pytorch_lightning/strategies/deepspeed.py

```diff
@@ -29,6 +29,17 @@
 from torch.optim import Optimizer
 
 import pytorch_lightning as pl
+from lightning_lite.utilities.distributed import (
+    _get_process_group_backend_from_env,
+    get_default_process_group_backend_for_device,
+    log,
+)
+from lightning_lite.utilities.enums import AMPType, PrecisionType
+from lightning_lite.utilities.optimizer import optimizers_to_device
+from lightning_lite.utilities.rank_zero import rank_zero_deprecation, rank_zero_info, rank_zero_warn
+from lightning_lite.utilities.seed import reset_seed
+from lightning_lite.utilities.types import _LRScheduler, _PATH, ReduceLROnPlateau
```

Comment:
I do not think we shall be using protected even if they come from our own package
in such case, lets make it public or keep it in PL